dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -0,0 +1,160 @@
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { remSpace, textSans } from '@guardian/source-foundations';
+import {
+	SvgCheckmark,
+	SvgNotificationsOff,
+	SvgNotificationsOn,
+	SvgPlus,
+} from '@guardian/source-react-components';
+import { decidePalette } from '../lib/decidePalette';
+
+type IconProps = {
+	isFollowing: boolean;
+	format: ArticleFormat;
+	iconIsFollowing: EmotionJSX.Element;
+	iconIsNotFollowing: EmotionJSX.Element;
+};
+
+const FollowIcon = ({
+	isFollowing,
+	format,
+	iconIsFollowing,
+	iconIsNotFollowing,
+}: IconProps): EmotionJSX.Element => (
+	<div
+		css={css`
+			background-color: ${isFollowing
+				? decidePalette(format).fill.shareIcon
+				: // TODO use decidePalette
+				  'white'};
+			height: 24px;
+			width: 24px;
+			margin: 0 0.3ch 0 0.5ch;
```

Comment:
I'm not that familiar with the `ch` unit, and we don't use it much in DCR from what I can tell. Do you have a strong preference for using it over something like `rem`?

Reply:
I think this was a copy/paste from AR which used ch. I will change.